src/components/ListView/ListView.js

```diff
@@ -180,17 +183,24 @@ export class ListView extends Component {
       RESULT_COUNT_INCREMENT,
       initialValues,
       ENTITY_KEY,
+      withNewRecordButton,
+      detailProps,
       visibleColumns,
       columnWidths,
       RecordView,
       RecordForm,
       renderHeaders,
+      actionMenuItems,
     } = this.props;
     const { showRestoreModal } = this.state;
 
     const urlQuery = queryString.parse(search);
     const searchTerm = trimSearchTerm(urlQuery.query);
 
+    const actionMenu = Array.isArray(actionMenuItems) && !!actionMenuItems.length
```

Comment:
I see this chunk repeating in other profiles. I am thinking about simplifying it or more to utils. I am thinking about 
`!isEmpty(actionMenuItems)` util from `lodash`. Please check if it does make sense,